tldw_Server_API/app/services/admin_guardrails_service.py

```
@@ -15,13 +18,81 @@ def _enterprise_admin_mode_enabled() -> bool:
+        exp_dt = datetime.fromtimestamp(raw_exp, tz=timezone.utc)
+    elif isinstance(raw_exp, datetime):
+        exp_dt = raw_exp if raw_exp.tzinfo is not None else raw_exp.replace(tzinfo=timezone.utc)
+
+    if jti and exp_dt is not None:
+        blacklist = get_token_blacklist()
+        actor_id = int(actor["id"]) if isinstance(actor, dict) else None
+        await blacklist.revoke_token(
+            jti=jti,
+            expires_at=exp_dt,
+            user_id=actor_id,
+            token_type="admin_reauth",
```

Ruff (0.15.6)

[error] Possible hardcoded password assigned to argument: "token_type"

(S106)